CONTRIBUTING.md

```diff
@@ -221,14 +223,14 @@ Run the following steps:
 1. Optional: If you like to use your custom favicon, please copy it to `src/favicon.ico`
 1. Optional: If you like to use your custom theme, replace the file `src/custom-theme.scss`.
    You can generate custom themes [here](http://mcg.mbitson.com/)
-1. Copy the file `src/environment.ts` to `src/environment.dev.ts` and adjust the values.
+1. Copy the file `src/environments/environment.ts` to `src/environments/environment.dev.ts` and adjust the values. In order to access the backend, set `backend_url` to your backend (defaults to "http://localhost:8000/api/v1").
 1. Run the frontend with:
 
    ```sh
    make dev
    ```
 
-1. You should see the frontend running von port 4200.
+1. You should see the frontend running from port 4200.
```

Comment:
```suggestion
1. You should see the frontend running on port 4200.
```

Thanks for fixing the typo. `on` is a bit more beautiful in my eyes, what do you say?

Reply:
I though it was the german word “von” and not a typo on “on”, so no problem, I put back “on”.